packages/components/src/popover/utils.ts

```diff
@@ -206,8 +228,9 @@ export const getReferenceElement = ( {
 	} else if ( fallbackReferenceElement ) {
 		// If no explicit ref is passed via props, fall back to
 		// anchoring to the popover's parent node.
-		referenceElement = fallbackReferenceElement.parentNode;
+		referenceElement = fallbackReferenceElement.parentElement;
```

Comment:
Runtime change: using `parentElement` instead of `parentNode`, since `Node`s don't have the `getBoundingClientRect()` function and therefore can't be used as the popover's reference (anchor)